CONTRIBUTING.md

```diff
@@ -0,0 +1,655 @@
+
+**1. Navigate to the page from within GitHub**
+
+Start at the [root of the content tree (the `/src` directory)](/src) and navigate down to the page you want to edit.  For example, if you want to edit the page at https://galaxyproject.org/community/ you would navigate:
+
+[`/src`](/src) &rarr; [`/src/community`](/src/community)
+
+Within the `community` directory you will see a few image files, and a file named `index.md`.  *Click* on it.
+
+**2. Use the Edit link on any Galaxy Hub page.**
+
+You can jump from any page in the Galaxy Hub directly to its source `index.md` file in GitHub.  Every Galaxy Hub web page has an **Edit** link in the upper right hand corner, that goes to that web page's `index.md` file in GitHub.
+
+## Edit the page in Github
+
+To edit an `index.md` file in GitHub, *click* the **pencil icon**. (The icon only appears if you are logged in.) When you hover over the icon, GitHub displays
+
+> Fork this project and edit the file.
+
+More on that in a bit.  Clicking the pencil icon takes you to a web form where you can edit the page text. The top of the edit page says:
+
+> You’re editing a file in a project you don’t have write access to. We’ve created a fork of this project for you to commit your proposed changes to. Submitting a change to this file will write it to a new branch in your fork, so you can send a pull request. 
+
+GitHub has now given you your own copy of the `galaxy-hub` repository *and you are now editing that copy under your account.*  Edit the text of the page in the web form.  There is a **Preview Changes** tab you can use to review your edits before you commit them.
+
+Once your edits are in good shape commit them by providing a short (and optionally, a longer) description of the change at the bottom of the form.
+
+*Click* **Propose file change** at the bottom of the form.  The commits the changes to *your* copy of the `galaxy-hub` repo. ("repo" = "repository" in GitSpeak).
+
+### What if I mess it up?  Are there guidelines?
+
+Thanks for your concern!  But, *we got this:*
+
+1. Until you get the hang of this ( *see What happens next and becoming an Editor* below) all of your edits will be reviewed by an experienced editor before they go to the web site.
+2. There are guidelines below, but don't worry about them too much when you start.  Just focus on the content you are fixing or adding. If you really want to "do the right thing" focus on the basic guidelines first, and then expand out as you need to.
+3. **Be bold.** We mean it.
+
+### You mentioned guidelines?
+
+Guidelines are distributed throughout this document.  Each is tagged with one of these icons:
+
+- :hatching_chick:  - A basic guideline, represented by a hatching chick.  Try to follow these from the start.
```

Comment:
you still have both - the bullet list of chicks and the table of chicks

Reply:
Which representation is better?  

I like the list, but am willing to convert everything to tables if the consensus is otherwise.

Reply:
+1 on the bullets

Reply:
Dropping replacing tables.